calendar_af/i18n/sv_SE.po

```diff
@@ -1072,16 +1090,21 @@ msgstr "Startdatum för en tillfälle"
 msgid "Start of repeat"
 msgstr "Start datum"
 
+#. module: calendar_af
+#: code:addons/calendar_af/wizard/create_local_occasion.py:158
+#, python-format
+msgid "Start or stop time is not and exacly an hour or halfhour."
```

Comment:
The place where this originates should be fixed.
"Start or stop time is not **and** exacly an hour or halfhour." - And should be removed?